src/utils/tablify.js

```diff
@@ -42,10 +42,15 @@ const getKeyValueObject = object => {
 		.join('\n');
 };
 
+const getKeyValueArray = array =>
+	array.some(item => typeof item !== 'object')
```

Comment:
Surely we want to use `getKeyValueObject` if any of the members are objects even if some aren't?

Reply:
At this point, we expect all the elements of array have the same shape

Reply:
Why not make it future-proof though?